src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -37,7 +37,7 @@ const useStyles = makeStyles((theme) => ({
   },
   stats: {
     position: 'absolute',
-    color: 'white',
+    color: '#e5e5e5',
```

Comment:
If our intention is to add a theme, we shouldn't be hardcoding in hex codes for the colours. I see that you added this hex code in `theme.js`. Instead of `color: '#e5e5e5'`, you can do `color: theme.palette.background.default`, this will apply the colour from `theme.js`